### PR TITLE
docs(fr): translate the whole guides

### PR DESCRIPTION
**Pull request by kissu**
Here is the current status of the French 🇫🇷 translated locales.
I will do it in the same order as the regular english guides. 👍🏻 
Related to the issue #550.

#### Current second read situation:
- [x] get started
- [x] concepts
- [x] features
- [ ] directory structure (half-done)
- [ ] configuration glossary (half-done)
- [ ] internals glossary
- [ ] components glossary

### Misc stuff 📎 
  - [ ]  give it a second read 👀 
  - [x]  double check that sub-sections (like `Directory Structure`) are properly translated in the locale, maybe try to build the project ? 🇬🇧 is used during `nuxt dev`
  - [x]  not sure where are the questions and how they do work, but better to double-check on that one too when merging this translation
  - [x]  i18n file
  - [x]  i18n block on nuxt.config

### Get started 🚗 
  - [x]  installation.md
  - [x]  routing.md
  - [x]  directory-structure.md
  - [x]  conclusion.md
  - [x]  upgrading.md

### Concepts 📖 
  - [x]  views.md
  - [x]  context-helpers.md
  - [x]  server-side-rendering.md
  - [x]  static-site-generation.md
  - [x]  nuxt-lifecycle.md

### Features 🎨
  - [x]  rendering-modes.md
  - [x]  deployment-targets.md
  - [x]  file-system-routing.md
  - [x]  data-fetching.md
  - [x]  meta-tags-seo.md
  - [x]  configuration.md
  - [x]  loading.md
  - [x]  nuxt-components.md
  - [x]  transitions.md
  - [x]  live-preview.md

### Directory structure 🌵
  - [x]  nuxt.md
  - [x]  assets.md
  - [x]  components.md
  - [x]  content.md
  - [x]  dist.md
  - [x]  layouts.md
  - [x]  middleware.md
  - [x]  modules.md
  - [x]  pages.md
  - [x]  plugins.md
  - [x]  static.md
  - [x]  store.md
  - [x]  nuxt-config.md

### Configuration glossary 🔧 
  - [x]  configuration-build.md
  - [x]  configuration-builddir.md
  - [x]  configuration-cli.md
  - [x]  configuration-css.md
  - [x]  configuration-components.md
  - [x]  configuration-dev.md
  - [x]  configuration-dir.md
  - [x]  configuration-env.md
  - [x]  configuration-extend-plugins.md
  - [x]  configuration-generate.md
  - [x]  configuration-global-name.md
  - [x]  configuration-head.md
  - [x]  configuration-hooks.md
  - [x]  configuration-ignore.md
  - [x]  configuration-loading.md
  - [x]  configuration-loading-indicator.md
  - [x]  configuration-mode.md
  - [x]  configuration-modern.md
  - [x]  configuration-modules.md
  - [x]  configuration-modulesdir.md
  - [x]  configuration-plugins.md
  - [x]  configuration-render.md
  - [x]  configuration-rootdir.md
  - [x]  configuration-router.md
  - [x]  configuration-runtime-config.md
  - [x]  configuration-server.md
  - [x]  configuration-server-middleware.md
  - [x]  configuration-srcdir.md
  - [x]  configuration-target.md
  - [x]  configuration-telemetry.md
  - [x]  configuration-transition.md
  - [x]  configuration-vue-config.md
  - [x]  configuration-watch.md
  - [x]  configuration-watchers.md

### Internals glossary 🌐 
  - [x]  context.md
  - [x]  $nuxt.md
  - [x]  internals.md
  - [x]  internals-module-container.md
  - [x]  internals-nuxt.md
  - [x]  internals-renderer.md
  - [x]  internals-builder.md
  - [x]  internals-generator.md
  - [x]  nuxt.md
  - [x]  nuxt-render.md
  - [x]  nuxt-render-route.md
  - [x]  nuxt-render-and-get-window.md

### Components glossary 👓 
  - [x]  pages-fetch.md
  - [x]  pages-head.md
  - [x]  pages-key.md
  - [x]  pages-layout.md
  - [x]  pages-loading.md
  - [x]  pages-middleware.md
  - [x]  pages-scrolltotop.md
  - [x]  pages-transition.md
  - [x]  pages-validate.md
  - [x]  pages-watchquery.md

### CODE REVIEW

**Comment by kissu** [comment]
Things done besides my own commits:
- merged upstream `master`
- resolved conflicts
- merged #752 translations
- updated the todo accordingly 👌🏻 

NB: The force push was basically a way to reset the `cherry-pick` 🍒 that failed pretty hard since Github likes to put the whole `master` changes into this PR as explained [here](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch), so annoying... Still managed to rollback it and fix it properly. Sorry for that (this _feature_ is my Nemesis !).

I've probably misspelled some words, have sub-par translations on some sections but I will give it a re-read to fix those (and probably strip some remaining 🇬🇧 sentences too) before the final merge.

I'm still waiting on people's opinion on [that question](https://github.com/nuxt/nuxtjs.org/issues/550#issuecomment-697413144). 🙏🏻 😄 

Still some work to do, **gambatte** ! 💪🏻

**Comment by kissu** [comment]
Ok, I still do have other commits from the base branch as stated [here](https://github.com/isaacs/github/issues/750#issuecomment-602064173), it's a feature apparently. 🤷🏻 

So yeah, I kinda give up on that one, it will maybe hide useless commits (the one **already** merged into the upstream `master`) once the PR is open or idk.

Remaining stuff is to give a quick second read. Will keep this message updated on where this process is currently at.

**Comment by debs-obrien** [comment]
hey @kissu great work. almost there. I would remove the en files from your pr so that it only adds french translation. Any improvements to en folder should be in separate pr. Thanks

File: content/fr/guides/configuration-glossary/configuration-transition.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ La clé de transition dans `nuxt.config.js` est utilisée pour définir les prop
 
 - Type: `String` ou `Object`
 
-> Utilisé pour définir les propriétés par défaut des transitions de la mise en page. La valeur fournie dans l'option `name` est configurée pour fonctionner avec le nom fourni dans `layout` de votre dossier `layouts`.
+> Utilisé pour définir les propriétés par défaut des transitions de la mise en page. La valeur fournie dans l'option `name` est configurée pour fonctionner avec le nom fourni dans `layout` de votre répertoire `layouts`.
```
Comment (roiLeo) [inline content/fr/guides/configuration-glossary/configuration-transition.md:46]: votre répertoire -> notre répertoire ?

File: content/fr/guides/directory-structure/assets.md [hunk 2]
```diff
@@ -239,18 +237,18 @@ Pour ces deux loaders, la configuration par défaut est la suivante:
 
 Ce qui veut dire que n'importe quel fichier en dessous de 1 Ko sera mis sur une seule ligne de data URL encodé en base-64. Autrement, l'image/police de caractères sera copiée dans le répertoire correspondant (à l'intérieur du répertoire `.nuxt`) avec un nom comprenant le hash d'une version pour une meilleure mise en cache.
 
-Lorsqu'on lancera l'application avec un `nuxt`, avec un template tel que `pages/index.vue`:
+Lorsque on lancera l'application `nuxt` avec un template tel que `pages/index.vue`:
```
Comment (roiLeo) [inline content/fr/guides/directory-structure/assets.md:240]: "Lorsqu'on" ou "Lorsque l'on" mais pas "Lorsque on"

**Comment by kissu** [comment]
I do not have a lot of time these days, so I guess we will finish this PR.

Will come back some time later on to give a second-read to the various sections. I rather see it published right now since Debbie is on a huge rhythm with her videos. The result will be good enough still ! 😎

File: content/fr/guides/directory-structure/store.md [hunk 3]
```diff
@@ -190,9 +190,9 @@ export default {
 </style>
 ```
 
-La méthode des modules fonctionne également pour les définitions de haut niveau sans implémenter un sous-répertoire dans le dossier store.
+La méthode des modules fonctionne également pour les définitions de haut niveau sans implémenter un sous-répertoire dans le répertoire `store`.
 
-Exemple pour l'état `state` : vous créez un fichier `store/state.js` et vous ajoutez ce qui suit.
+Exemple pour l'état `state`: nous créons un fichier `store/state.js` et nous ajoutons ce qui suit.
```
Comment (roiLeo) [inline content/fr/guides/directory-structure/store.md:195]: add whitespace after `state` and before ':'

**Comment by debs-obrien** [comment]
hey @kissu let me know when this is ready to go live and be merged.

**Comment by kissu** [comment]
It is actually ready, sorry should have been more explicit ! 😅

**Comment by debs-obrien** [comment]
haha, yes I never saw last message. sorry.

**Comment by debs-obrien** [comment]
amazing work by the way. thanks so much for all your contributions. really appreciated.

**Comment by kissu** [comment]
Was a pleasure to help ! 🙏🏻 
No worries, will continue to contribute to Nuxt if it's doable with my current skill level. 😸

**Comment by debs-obrien** [comment]
great. yes thanks also @roiLeo, very much appreciated

**Comment by debs-obrien** [comment]
will be live when this pr is merged: https://github.com/nuxt/nuxtjs.org/pull/992

**Pull request merged**